### PR TITLE
fix: deletion of items that don't implement truthiness

### PR DESCRIPTION
**Pull request by JeppeKlitgaard**
Example: pandas.DataFrame or pandas.Series.

The current memory backend implementation evaluates truthiness on deletion from the cache.

This works out most of the time but results in an error when caching pandas.Dataframe or pandas.Series objects.

### CODE REVIEW

**Comment by argaen** [comment]
Hey @JeppeKlitgaard would you mind adding tests for this please?

File: aiocache/decorators.py [hunk 1]
```diff
@@ -183,12 +183,20 @@ def __init__(self, lease=2, **kwargs):
         super().__init__(**kwargs)
         self.lease = lease
 
-    async def decorator(self, f, *args, **kwargs):
+    async def decorator(self,
+        f,
+        *args,
+        cache_read=True,
+        cache_write=True,
+        aiocache_wait_for_write=True,
```
Comment (argaen) [inline aiocache/decorators.py:191]: these changes seem unrelated with what you propose, can you remove them please?

Comment (bbelyeu) [reply aiocache/decorators.py:191]: Agreed. The cache_read and cache_write should be a new issue and a different MR if you'd like to propose that functionality. It's a different discussion (feature add) as opposed to the original bug fix.
